content/cpp/concepts/queues/terms/front/front.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: '.front()'
+Description: 'Returns the next element within the queue.'
+Subjects:
+  - 'Computer Science'
+  - 'Game Development'
+Tags:
+  - 'Objects'
+  - 'OOP'
+  - 'Classes'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The `.front()` method returns the next element within the queue. This will be the "oldest" element of the queue, the element added the earliest.
```

Comment:
```suggestion
The `.front()` method returns the next element within the queue. This will be the oldest element of the queue or the element which was added the earliest.
```